crates/database/src/fee_policies.rs

```diff
@@ -0,0 +1,175 @@
+use {
+    crate::{auction::AuctionId, OrderUid, PgTransaction},
+    sqlx::PgConnection,
+    std::ops::DerefMut,
+};
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct FeePolicy {
```

Comment:
can we kill this struct and just implement conversion from `autopilot::domain::fee::Policy` to `FeePolicyRow` directly and put that into `crates/autopilot/src/infra/persistence/dto/fee_policy.rs`

Reply:
Sure, we can, but then we will have only `domain::fee::Policy` without `auction_id` and `order_uid` fields.